ledger/ledger.go

```diff
@@ -69,16 +68,25 @@ type Ledger struct {
 	headerCache heapLRUCache
 }
 
+// InitState structure defines blockchain init params
+type InitState struct {
+	InitBlocks   []bookkeeping.Block
```

Comment:
Drop the "Init" prefix from the variable, since it's already reflected in the data structure name.
No need for array of blocks.

Reply:
Removed Init, switching from an array of blocks to a single block seemed to large, will do as a separate PR